spec/lib/mastodon/redis_configuration_spec.rb

```diff
@@ -45,6 +45,20 @@
       it 'uses the url from the base config' do
         expect(subject[:url]).to eq 'redis://localhost:6379/0'
       end
+
+      context 'when the base config uses sentinel' do
+        around do |example|
+          ClimateControl.modify REDIS_SENTINELS: '192.168.0.1:3000,192.168.0.2:4000', REDIS_SENTINEL_MASTER: 'mainsentinel' do
+            example.run
+          end
```

Comment:
Would also be worth checking:

```
REDIS_SENTINELS: '192.168.0.1,192.168.0.2:4000'
```

So mixed, one with port, one without.

Reply:
I added this in the PR mentioned above.